services/user/Supervisor/ui/src/plugin/pluginHost.ts

```diff
@@ -215,16 +227,66 @@ export class PluginHost implements HostInterface {
     getAppUrl(app: string): string {
         return `${siblingUrl(null, app)}`;
     }
+    
+    private setActiveUserPrompt(subpath: string | undefined, upPayloadJsonStr: string): Result<string, RecoverableErrorPayload> {
```

Comment:
I'm trying to understand this better. Are we not worried about awaiting these calls and any unhandled promise rejection errors in this method?